packages/modeling/src/operations/hulls/hullChain.js

```diff
@@ -30,12 +31,16 @@ import { hull } from './hull.js'
  *       +-------+                +-------+
  */
 export const hullChain = (...geometries) => {
```

Comment:
I just noticed... can hullChain() handle path2 geometries?

Reply:
It can. I updated jsdoc to match behavior